README.md

```diff
@@ -27,3 +29,16 @@ To make the bug report accurate and easy to understand, please try to create iss
 * _Scoped_. One bug per issue; please do not follow up with another bug inside one report. It may be worthwhile to read [Elika Etemad’s article on filing good bug reports](http://fantasai.inkedblade.net/style/talks/filing-good-bugs/) before creating a bug report.
 
 We might ask for further information to locate a bug. A duplicated bug report will be closed.
+
+#### Tectonic Installer Progress file
+
+Click the *Save progress* button at the top right corner of any Installer screen to download a snapshot of install configuration and progress. A JSON-formatted file named `tectonic-progress` will be written beneath the browser's download path. Including the Progress file from a failed installation can assist troubleshooting. Attach the Progress file to the corresponding Issue in this repository.
+
+Before sending a Progress file, ensure that the potentially sensitive values of the following keys are removed or obscured from the Progress file:
+
```

Comment:
`adminPassword` too, right?

There's just so much sensitive data there. Are we sure we want to do this? cc @sym3tri